### PR TITLE
Android media session api

### PR DESCRIPTION
**Pull request by stonko1994**
## Description

Add sample how to use `MediaSessionAPI` for Chrome on Android.

### CODE REVIEW

File: mobile/androidMediaSessionAPI.html [hunk 1]
```diff
@@ -0,0 +1,232 @@
+<!DOCTYPE html>
+<!--
+*
+* Copyright (C) 2015, bitmovin GmbH, All Rights Reserved
```
Comment (bitmovin-kenny) [inline mobile/androidMediaSessionAPI.html]: that copyright notice seems a bit outdated

File: mobile/androidMediaSessionAPI.html [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+*
+* Created on: 2015-07-25 11:35:04
```
Comment (bitmovin-kenny) [inline mobile/androidMediaSessionAPI.html]: This date is wrong, i don't know if we actually need it, maybe its better to just remove the autor + created

File: mobile/androidMediaSessionAPI.html [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+                <p>For more information about the bitmovin player, please have a look at our online <a
+                        href="//bitmovin.com/support" target="_blank">Developer Section</a>.</p>
+            </div>
+        </div>
+    </div>
+</div>
+<script type="text/javascript">
+
+    if (location.protocol === 'file:') {
+        document.getElementById('webserver-warning').style.display = 'block';
```
Comment (bitmovin-kenny) [inline mobile/androidMediaSessionAPI.html]: There is no element with the id 'webserver-warning', so this code is faulty.
There are also css rules for this id.
Either you add an element or you just drop all of the functionality regarding this id, maybe @dweinber knows why we treat the file protocol error in a special way in the demos and if this still makes sense

File: mobile/androidMediaSessionAPI.html [hunk 4]
```diff
@@ -0,0 +1,232 @@
...
+
+    var conf = {
+        key: 'YOUR KEY HERE',
+        source: {
+            dash: 'https://bitdash-a.akamaihd.net/content/MI201109210084_1/mpds/f08e80da-bf1d-4e3d-8899-f0f6155f6efa.mpd',
+            hls: 'https://bitdash-a.akamaihd.net/content/MI201109210084_1/m3u8s/f08e80da-bf1d-4e3d-8899-f0f6155f6efa.m3u8',
+            progressive: 'https://bitdash-a.akamaihd.net/content/MI201109210084_1/MI201109210084_mpeg-4_hd_high_1080p25_10mbits.mp4',
+            poster: 'https://bitdash-a.akamaihd.net/content/MI201109210084_1/poster.jpg'
+        },
+        style: {
```
Comment (bitmovin-kenny) [inline mobile/androidMediaSessionAPI.html]: this style config can be omitted, as it is the default

File: mobile/androidMediaSessionAPI.html (hunk 1 above)
Comment (stonko1994) [reply mobile/androidMediaSessionAPI.html]: should we just remove it? There are other samples where there is no copyright notice

Comment (dweinber) [reply mobile/androidMediaSessionAPI.html]: Just noticed we don't have a `LICENSE` file in this repo, will add one. You can drop the header, please do this then for the other samples as well (in another PR)

**Pull request merged**